.circleci/config.yml

```diff
@@ -84,27 +84,131 @@ jobs:
+            make develop
+            python benchmarks/huggingface.py --accuracy --ci --backend=aot_eager -d cuda --output=aot_eager_hf_inference.csv
+      - store_artifacts:
+          path: aot_eager_hf_inference.csv
+      - run:
+          name: Huggingface AotAutograd Eager inference result check
+          command: |
+            source .circleci/setup_env.sh
+            python .circleci/check_csv.py -f aot_eager_hf_inference.csv
+
+  aot_eager_timm_inference:
     machine:
-      # https://circleci.com/docs/2.0/configuration-reference/#available-linux-gpu-images
       image: ubuntu-2004-cuda-11.4:202110-01
     resource_class: gpu.nvidia.large
     steps:
       - checkout
       - install_deps
       - run:
-          name: TorchBench AotAutograd Eager run
+          name: TIMM AotAutograd Eager inference run
           command: |
             source .circleci/setup_env.sh
             make develop
-            python benchmarks/torchbench.py --accuracy --ci --skip-accuracy-check --training --backend=aot_eager -d cuda --output=aot_eager.csv
+            python benchmarks/timm_models.py --accuracy --ci --backend=aot_eager -d cuda --output=aot_eager_timm_inference.csv
       - store_artifacts:
-          path: aot_eager.csv
+          path: aot_eager_timm_inference.csv
       - run:
-          name: TorchBench AotAutograd Eager compiler accuracy
+          name: TIMM AotAutograd Eager inference result check
           command: |
             source .circleci/setup_env.sh
-            python .circleci/check_csv.py -f aot_eager.csv
+            python .circleci/check_csv.py -f aot_eager_timm_inference.csv
+
+  aot_eager_torchbench_training:
+    machine:
+      image: ubuntu-2004-cuda-11.4:202110-01
+    resource_class: gpu.nvidia.large
+    steps:
+      - checkout
+      - install_deps
+      - run:
+          name: TorchBench AotAutograd Eager training run
+          command: |
+            source .circleci/setup_env.sh
+            make develop
+            python benchmarks/torchbench.py --accuracy --ci --training --use-eval-mode --amp --backend=aot_eager -d cuda --output=aot_eager_torchbench_training.csv
+      - store_artifacts:
+          path: aot_eager_torchbench_training.csv
+      - run:
+          name: TorchBench AotAutograd Eager training result check
+          command: |
+            source .circleci/setup_env.sh
+            python .circleci/check_csv.py -f aot_eager_torchbench_training.csv
+
+  aot_eager_hf_training:
+    machine:
+      image: ubuntu-2004-cuda-11.4:202110-01
+    resource_class: gpu.nvidia.large
+    steps:
+      - checkout
+      - install_deps
+      - run:
+          name: Huggingface AotAutograd Eager training run
+          command: |
+            source .circleci/setup_env.sh
+            make develop
+            python benchmarks/huggingface.py --accuracy --ci --training --use-eval-mode --amp --backend=aot_eager -d cuda --output=aot_eager_hf_training.csv
```

Comment:
You can remove `--use-eval-mode`. `--accuracy` sets it up.

Reply:
Good to know. Let me do that next time when I update this file.